### PR TITLE
use mongosh connection to ocrd-database for job infos

### PR DESCRIPTION
**Pull request by bertsky**
first attempt (still using bash for everything, so DB access only via mongosh)

It looks like when you restart a job for the same workspace, the new active job is not shown because internally it gets confused with the previous job for that workspace – so we probably need to redefine the index.

### CODE REVIEW

**Review by markusweigelt — COMMENTED** [review]
Looks good too. I can only perform a functional test after my vacation.

File: Dockerfile [hunk 1]
```diff
@@ -60,6 +60,9 @@ RUN pip install ocrd
 # install mets-mods2tei and page-to-alto
 RUN pip install mets-mods2tei
 RUN pip install ocrd-page-to-alto
+# install mongosh
+RUN wget https://downloads.mongodb.com/compass/mongodb-mongosh_1.10.1_amd64.deb
```
Comment (markusweigelt) [inline Dockerfile:64]: Does the `mongosh` version even match the MongoDB version if it is not configurable externaly?

Comment (bertsky) [reply Dockerfile:64]: oh, good question!

We simply use `mongo:latest` image in the Monitor, and it says it's v6.0.6. I don't know how the versioning of the mongosh relates to that. I guess a CLI must support multiple versions anyway...

**Comment by bertsky** [comment]
039567a adds a simple web server (via `socat` and [sampo](https://github.com/jacobsalmela/sampo/pull/25)).

There are 3 endpoints:

Usage is simple: just convert any of the [command-line args](/slub/ocrd_manager/#processing) to CGI query options, e.g.

    curl "http://localhost:4004/for_presentation/testdata-presentation/mets.xml?url-prefix=https://digital.slub-dresden.de/data/kitodo&workflow=/workflows/ocr-workflow-default.sh"

this will automatically translate to

    for_presentation.sh --workflow /workflows/ocr-workflow-default.sh --url-prefix https://digital.slub-dresden.de/data/kitodo testdata-presentation/mets.xml

We now should document this, add some more logging, and start with a call-back interface (see #64).

**Comment by BartChris** [comment]
@bertsky I tried out this feature and encountered the problem that constructing the JSON for Mongo fails as there seems to be linebreaks in the JSON. Maybe i introduced some problems while a was copying the logic, so that some unwanted line breaks got injected? 
https://github.com/slub/ocrd_manager/blob/52587e9e2c3e669bf832da3d4839a7254d1327fe/ocrd_lib.sh#L71

<pre>
terminating with error $?=1 from HOME=/tmp mongosh --quiet --norc --eval "use ocrd" --eval "db.OcrdJob.insertOne( {#012           pid: $PID,#012           time_created: ISODate(\"$(date --rfc-3339=seconds)\"),#012           process_id: \"$PROCESS_ID\",#012           task_id: \"$TASK_ID\",#012           process_dir: \"$PROCESS_DIR\",#012           workdir: \"$WORKDIR\",#012           remotedir: \"$REMOTEDIR\",#012           workflow_file: \"$WORKFLOW\",#012           controller_address: \"$CONTROLLER\"#012      } )" $DB_CONNECTION on line 83 /usr/bin/ocrd_lib.sh
</pre>

For the moment i formatted it in a way to have everything in one line.

**Pull request merged**